<launch>
    <include file="$(find xycar_motor)/launch/xycar_motor.launch" />
    <include file="$(find xycar_lidar)/launch/lidar_noviewer.launch" />

    <!--node name="usb_cam" output="screen" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device" value="/dev/videoCAM" />
        <param name="autoexposure" value="false" />
        <param name="exposure" value="100" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap" />
    </node-->
    <node name="Object_Detection_System" pkg="ObjectDetectionSystem" type="ObjectDetectionSystem_node" output="screen"/>
    <param name="config_path" type="str" value="$(find ObjectDetectionSystem)/config/config.yaml"/>
    <!--node name="rosbag" pkg="rosbag" type="play" args="$(find ObjectDetectionSystem)/src/track.bag"/-->
</launch>
